powersimdata/data_access/data_access.py

```diff
@@ -196,6 +195,18 @@ def makedir(self, relative_path):
         target = os.path.join(self.root, relative_path)
         os.makedirs(target, exist_ok=True)
 
+    def remove(self, target, recursive=False):
+        """Remove target using rm semantics
+
+        :param str target: path to remove
+        :param bool recursive: delete directories recursively
+        """
+        if recursive:
+            target = os.path.join(target, "**")
+        files = glob.glob(target, recursive=recursive)
```

Comment:
According to the documentation, it seems `glob` is used for Unix shell only?

Reply:
I believe "unix style" is just a reference to the origin. There are separate sections in the docs for [windows](https://docs.python.org/3/library/windows.html) and [unix](https://docs.python.org/3/library/unix.html) specific features. Will do a test to confirm though.

Reply:
Looks like there could be some inconsistencies (see this [post](https://python-forum.io/Thread-Version-of-glob-for-that-Supports-Windows-Wildcards)) but it _should_ work for our current use case.

Reply:
Thanks for looking into that. I think it should be fine as long as it works for our current use case. As said, we can always revisit it in the future if our usage goes beyond the available scope.

Reply:
If this fails, it seems like we can use regex workarounds. I don't think we use any `[` or `]` characters _within_ our data directories, so hopefully any of those characters in parent directories won't be a problem.